lib/resources/datav2/websocket.ts

```diff
@@ -76,8 +77,10 @@ interface WebsocketSession {
   backoffIncrement: number;
   url: string;
   currentState: STATE;
-  pingTimeout?: NodeJS.Timeout;
-  pingTimeoutThreshold: number;
+  pongTimeout?: NodeJS.Timeout;
+  pingInterval?: NodeJS.Timer;
+  pongWait: number;
+  pingCounter: number;
```

Comment:
Why is this needed?

Reply:
According to the standard sending payload in a ping message is optional. I almost removed it but the go's websocket library sends incrementing number and I decided doing the same. Should I remove?

Reply:
I think it's fine to remove it, the server is doing nothing with it.

Reply:
Removed